tests/registration_hw19/seccussUserRegistration.spec.js

```diff
@@ -0,0 +1,62 @@
+import {expect, test} from "@playwright/test";
+import {WelcomePage} from "../../src/homework19/WelcomePage/WelcomePage.js";
+import {USER_DATA} from "../../src/data/userData.js";
+import GaragePage from "../../src/homework19/GaragePage/GaragePage.js";
+import SettingsPage from "../../src/homework19/SettingsPage/SettingsPage.js";
+import {SideBarMenu} from "../../src/homework19/GaragePage/components/SideBarMenu.js";
+import RemoveUserPopup from "../../src/homework19/SettingsPage/components/RemoveUserPopup.js";
+
+
+test.describe("Registration new user", ()=> {
+
+    test.describe("Positive case", () => {
+        let popup
+
+        test.beforeEach(async ({page}) => {
+            const welcomePage = new WelcomePage(page);
+            await welcomePage.navigate();
+            popup = await welcomePage.openSignUpPopup();
+
+
+        })
+
+        test.afterEach(async ({page}) => {
+            const garagePage = new GaragePage(page);
+            const settingsPage = new SettingsPage(page);
+            const removeUserPopup = new RemoveUserPopup(page);
+            const sideBar = new SideBarMenu(page);
```

Comment:
do not create ainstance of SideBar in tests, move it to POM

Reply:
done